tests/e2e/support/objects/app-files/resource/actions.ts

```diff
@@ -227,8 +227,6 @@ export const downloadResources = async (args: downloadResourcesArgs): Promise<Do
           page.locator(downloadButtonSideBar).click()
         ])
 
-        await sidebar.close({ page: page })
-
```

Comment:
@ScharfViktor Instead of removing those steps, you can cherry pick this commit from my branch https://github.com/owncloud/web/pull/7910/commits/109f602f5cbc329fff4038d37b43ab595ccd9596 . (Only click the `visible` element - the test might be getting element from other panels)
https://github.com/owncloud/web/blob/109f602f5cbc329fff4038d37b43ab595ccd9596/tests/e2e/support/objects/app-files/utils/sidebar.ts#L55-L57

Reply:
Thanks for hint. let see what say CI